Project_source/app/src/main/java/android/clicker/school_live_simulator/NullComputerCourseState.kt

```diff
@@ -0,0 +1,20 @@
+package android.clicker.school_live_simulator
+
+open class NullComputerCourseState: ComputerCourseState {
+    override price: Int = 0
+
+    init {
+        available_web_task.add(WebTask.ACCOUNT_BOOST)
+    }
+
+    override fun buyNexCourse(student: Player) {
+        Player.courses = OnlineWorkCourseState()
+    }
```

Comment:
Same thing for every class
```suggestion
    override fun buyNexCourse(courses: Player.Courses) {
        courses.computer_course = OnlineWorkCourseState()
    }
```

Reply:
Fixed